### DISCUSSION TITLE
Problems occur in install Autogen Studio locally with the method of Install 2.

### DISCUSSION THREAD

**Discussion by Suhongshi** [microsoft/autogen#2416] [Q&A]
The second way of install Autogen Studio does not need "pip install autogenstudio" but when I follow the procedures untill I input "autogenstudio ui --port 8081" into my conda terminal，it shows 
“autogenstudio: command not found”.what should I do next?

**Comment by OKSHIJIAN** [comment]
metoo

**Comment by frankmunozjonel** [comment]
Be sure you have a C++ compiler installed on your machine. Easiest way is to install visual studio (not vs code), then install C++ dev/support from the visual studio. Once the C++ compiler is installed, your autogenstudio will be able to install all the correct pip dependencies and run the local server command. https://learn.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-170

**Comment by danybr0** [comment]
Encountered the same issue. Unfortunately, I wasn’t able to resolve it directly.

However, I found that running AutogenStudio via Anaconda works reliably.

✅ Recommendation: Install and run AutogenStudio using Anaconda for a smoother experience.

Refer to this link for more info:
https://microsoft.github.io/autogen/stable/user-guide/autogenstudio-user-guide/installation.html
